google/cloud/storage/tests/object_hash_integration_test.cc

```diff
@@ -192,8 +199,13 @@ TEST_F(ObjectHashIntegrationTest, WriteObjectExplicitEnable) {
   EXPECT_THAT(os.computed_hash(), HasSubstr(ComputeMD5Hash(LoremIpsum())));
   EXPECT_THAT(os.received_hash(), HasSubstr(ComputeMD5Hash(LoremIpsum())));
   if (meta->has_metadata("x_emulator_upload")) {
-    ASSERT_TRUE(meta->has_metadata("x_emulator_no_crc32c"));
-    ASSERT_TRUE(meta->has_metadata("x_emulator_no_md5"));
+    ASSERT_THAT(meta->metadata(), Contains(Pair("x_emulator_no_crc32c", _)));
```

Comment:
nit: s/ASSERT/EXPECT/

also consistency nit: in other tests, you put this check in each branch of `if (UsingGrpc())`